layouts/partials/footer.html

```diff
@@ -20,10 +20,13 @@
         {{ with .Site.Params.copyright }}
         <div class="row">
           <div class="col-12">
-            <small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>
+            <small class="text-white">&copy; {{ now.Year}} The cert-manager Authors</small>
           </div>
           <div class="col-12">
-            <small class="text-white">cert-manager was originally created by <a href="https://jetstack.io"><span id="jetstack-footer">{{ with resources.Get "icons/jetstack.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}</span></a></small>
```

Comment:
Do we still want to keep the "created by Jetstack" text? From what I understand, this _is_ permitted by LF

Reply:
I have looked at a few CNCF sites and didn't see any, removed it till we have clarity on it

Reply:
I'd like for it to remain. It's on other project sites (eg - [Linkerd](https://linkerd.io/), [Kyverno](https://kyverno.io/), [Kuma](https://kuma.io/). As @munnerz says, it's also permitted by the [website-guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md):

>It is OK to say that, e.g., “Prometheus was originally created by Soundcloud” or “Kubernetes builds upon 15 years of experience of running production workloads at Google,” but the origin company should not otherwise be referred to on the project homepage.

How about text that is similar to the message used by Kuma? I really like the sentiment, and it's of course very true for cert-manager and it's 250+ contributors.

> Kuma was originally created by Kong, Inc. and made successful by its open community.

Reply:
Thanks for the link! the guidelines in teh TOC ticket didn't mention it, happy to add it back as it was!